### ISSUE TITLE
Badge Step Time(s) over 1 not saving.

### ISSUE THREAD

**Issue by designerhacks** [opencredit/badgeos#502]
Currently whenever I save a Required step with more than the default "1" setting for time(s).  It doesn't save.  

When I click "Save All Steps" the wheel spins and it appears to save, but upon updating and refeshing the page, the time(s) shows "1".

![banners_and_alerts_and_edit_badges_ _designer_hacks_ _wordpress](https://cloud.githubusercontent.com/assets/16158867/11602206/7af76d5c-9aa6-11e5-8260-00842125b5be.jpg)

**Comment by tw2113** [comment]
Are you able to test this with all but the essential plugins deactivated? it's possible there's conflict coming from elsewhere

**Comment by designerhacks** [comment]
Michael,

I'll create a staging environment to test this.

**Comment by tw2113** [comment]
Appreciated to help narrow down scope.

**Comment by designerhacks** [comment]
Michael,

I've discovered that the issue occurs when the BadgeOS LearnDash Add-on is
activated.

With just BadgeOS and the LearnDash plugin active, there doesn't seem to be
any issues.

**Comment by tw2113** [comment]
I can't recall the logic behind setting things to "1" in the metabox field, and the person that likely would, isn't available right now.

Can you do me a quick favor and let me know your trigger setup for this Learndash version? I'm wondering if the meta data still saves with your input value, despite what the metabox is showing.

**Comment by designerhacks** [comment]
Micahel,  I'd be happy to help.  I'm not sure what you mean by trigger
setup or how to let you know what it is.  Sorry, my programming knowledge
is pretty limited.  Any direction on how to get this information to you
would be helpful.

**Comment by tw2113** [comment]
You have your achievement, which I assume is based on LearnDash activity. What I'm asking for is the various settings you have for that achievement. Steps required to earn it, etc.

**Comment by designerhacks** [comment]
The achievement I'm referring to is actually based on log in time(s).  The
time(s) issue occurs for all steps LearnDash or not.

**Comment by tw2113** [comment]
Then any of the info you have set for the achievements would be great so I
can try and recreate as matching as possible

**Comment by designerhacks** [comment]
Name: Repeat Defender
Points Awarded: 25
Earned By: Completing Steps
Sequential Steps: Not Checked
Show Earners: Not checked
Congratulations Text: We are the two best friends that anyone could have. And we will never ever ever ever ever leave each other.
Maximum Earnings: 1
Hidden?: Show to User
Required Steps:
1 Require Log in to Website 1 time(s).
Label: Login 50 times.
Send to Credly when earned: No

**Comment by designerhacks** [comment]
Settings:
![screenshot_120515_043230_pm](https://cloud.githubusercontent.com/assets/16158867/11610334/ca360690-9b6d-11e5-8500-2500ae2295be.jpg)

Plugins:
![screenshot_120515_043230_pm](https://cloud.githubusercontent.com/assets/16158867/11610333/c72737d0-9b6d-11e5-8170-e77d8c457075.jpg)

![screenshot_120515_043202_pm](https://cloud.githubusercontent.com/assets/16158867/11610331/b6811270-9b6d-11e5-96e7-1649f40e619c.jpg)

**Comment by tw2113** [comment]
Haven't forgotten, but you know, weekend. Will be trying to loop back around to this today.

**Comment by designerhacks** [comment]
I understand.  Thanks Michael.

**Comment by tw2113** [comment]
Still not managing to recreate. Entering 5 for the login amount and saving and it works fine. I am requesting an updated copy of LearnDash from the appropriate people, as I noticed I'm on 1.5.x and you're up on 2.1.x which actually be playing a factor here.

**Comment by designerhacks** [comment]
Any more luck with this?

**Comment by tw2113** [comment]
No, as per my last message, but I haven't done much with it since that last attempt either.

**Comment by wpaustria** [comment]
I got the same problem right here!
